src/sidebars/version-directive.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.version')
+        .value('version', '0.1.0')
```

Comment:
There's a grunt task that I've spotted that does this and also tags Git appropriately with releases, maybe it's worth using that?